docs/readmes/proposals/p027_magma_outbound_roaming_support.md

```diff
@@ -0,0 +1,136 @@
+
+## 2 Background
+
+### 2.1: Terminology
+
+**S6a** refers to the Authentication, Location, and Service definition interface of an HSS in a mobile core.
+**S8** refers to the control and user plane interfaces used when user data is home routed during an outbound roaming session.
+
+### 2.2 Outbound roaming
+
+Outbound roaming is when a subscriber gains access to services from a visited operator (VPLMN) other than their home network operator (HPLMN). In this case the HPLMN is a Magma deployment, and the VPLMN is another network operator with which this HPLMN has a roaming relationship.
+
+When outbound roaming, the standard interfaces are s6a and s8 which allow the VPLMN to authenticate the SIM credentials with the home network and route user traffic back to the home network. See diagram below from 3GPP TS 23.401 V17.3.0 (2021-12).
+
+![arch](https://user-images.githubusercontent.com/93994458/154709583-14f9bb52-486b-4908-b250-e521627b6f04.png)
+
+Currently, Magma supports only inbound roaming where magma is the VPLMN and another operator/core is the HPLMN. Magma supports the client side of both S6a and S8 interfaces.
+
+By implementing the server side of S6a and S8, this project will add support for outbound roaming to Magma.
+
+## 3 Implementation
+
+### 3.1 Diagrams
+
+The Roaming Gateway diagram shows the services which will run in the Roaming gateway. These services will support the user plane data transport for both inbound roaming GTP aggregation and outbound roaming when traffic is home routed.
+
+![roaming gateway](https://user-images.githubusercontent.com/93994458/154709967-7cd453ba-2070-4733-9e3f-dcc6f38624d2.png)
+
+Overall, when combined with the orchestrator and federation gateway, the proposed architecture is depicted below showing all key interfaces for inbound and outbound roaming.
+
+![arch](https://user-images.githubusercontent.com/93994458/154710530-f535e6c1-8221-4de4-9e7b-5fc974f2ee3d.png)
+
+The proposed architecture gives the Magma deploying operator the ability to locate the user plane carrying Roaming Gateway outside of AWS where data transport rates are more competitive and/or in a location favorable for connection to the GRX/IPX network.
+
+### 3.2 Scope of Change
+
+#### 3.2.1 Roaming Gateway Services
+
+The following existing services will be ported to run in the Roaming Gateway:
+
+- Health
+- S8_proxy
+- PolicyDB
+- SessionD
+- PipelineD
+- MobilityD
+- SubscriberDB
+- Control Proxy
+- Magmad
+- Eventd
+- td-agent-bit
+
+Similar to FeG and AGW, Roaming Gateway will be a full Magma gateway with control proxy connection to orchestrator and health service.
+
+##### 3.2.1.1 Health (modified)
+
+In addition to the health metrics added in the GTP Gateway proposal for inbound roaming, the following health metrics will also be added to health:
```

Comment:
`to health` - This is either an superfluous suffix or there is a word (e.g. `endpoint`) missing here.